debian/control

```diff
@@ -36,10 +36,8 @@ Build-Depends: debhelper (>= 11),
                libgtk-3-dev (>= 3.21.6),
                libibus-1.0-dev,
                libmutter-1-dev (>= 3.26.0),
-               libnm-glib-dev (>= 0.9.8) [linux-any],
-               libnm-glib-vpn-dev (>= 0.9.8) [linux-any],
-               libnm-gtk-dev (>= 0.9.8) [linux-any],
-               libnm-util-dev (>= 0.9.8) [linux-any],
+               libnm-dev (>= 1.2.0) [linux-any],
+               libnma-dev (>= 1.2.0) [linux-any],
```

Comment:
Do we really need the one for the network-manager-applet (libnma-dev)? I don't see why that would be needed (and Debian Buster seems to agree), but I might be missing something